long_term/test_load_long.py

```diff
@@ -0,0 +1,76 @@
+# pylint: skip-file
+
+"""Test file for load functions."""
+
+from unittest.mock import patch, MagicMock
+import pytest
+import pandas as pd
+from datetime import date
+from load_long import create_csv, get_s3_client, upload_csv, load_recordings
+
+
+class TestCreateCSV():
+
+    @patch("pandas.DataFrame.to_csv")
+    def test_create_csv_core_functionality(self, mock_to_csv):
+
+        mock_df = pd.DataFrame({"c1": [1, 2], "c2": [3, 4]})
+
+        expected_filename = f"summary-{date.today()}.csv"
+
+        assert create_csv(mock_df) == expected_filename
+
+    @patch("pandas.DataFrame.to_csv")
+    def test_create_csv_calls_to_csv(self, mock_to_csv):
+
+        mock_df = pd.DataFrame({"c1": [1, 2], "c2": [3, 4]})
+
+        res = create_csv(mock_df)
+
+        mock_to_csv.assert_called_once_with(res, index=False)
+
+    @patch("pandas.DataFrame.to_csv")
+    def test_create_csv_error_on_invlaid_input(self, mock_to_csv):
+
+        mock_df = "string"
+
+        with pytest.raises(TypeError) as err:
+            create_csv(mock_df)
```

Comment:
Why do you need to define mock_df here? Can't you just pass a string into the function?